prow/bugzilla/client.go

```diff
@@ -225,7 +225,17 @@ func (c *client) CloneBug(bug *Bug) (int, error) {
 	if err != nil {
 		return 0, fmt.Errorf("failed to get parent bug's comments: %v", err)
 	}
-	return c.CreateBug(cloneBugStruct(bug, comments))
+	id, err := c.CreateBug(cloneBugStruct(bug, comments))
```

Comment:
Is this re-entrant? If we fail to update and someone tries to do another `/bugzilla refresh` we will fail here as the cloned bug already exists, no?

Reply:
We identify bugs via the PR title. If we fail to clone and properly configure the bug, the bug number won't be added to the title. So I don't think that this should cause problems (other than a bug floating around in bugzilla). The user would have to do the cloning of the bug and retitling of the PR themselves in that case.

Also, this functionality isn't used by the `/bugzilla refresh` command (at least not yet). It only runs when a PR is opened with the correct description. We could also add functionality to handle on `/bugzilla refresh` instead, but that would be slightly more complicated (we would need to check whether the bug was created by the cherrypicker on every `/bugzilla refresh` and if yes, then identify what target release the bug has, creating a clone if it doesn't match the branch's target release).

Reply:
Ah, ok. I think we will get asked in the future to help devs make the cherry-pick automatically if we happen to fail on the first attempt, but I agree that as written this does not need to be re-entrant.